contributing/code_of_conduct_concrete_example_document.rst

```diff
@@ -0,0 +1,16 @@
+Code of Conduct: Concrete Example Document
+==========================================
+This is a living document that serves to give concrete examples to unwanted behaviour. These examples have all taken place somewhere in the PHP community in the past, and are clear code of conduct violations according to the Symfony code of conduct.
+
+Concrete Examples
+-----------------
+- Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment.
+- Deliberate misgendering or use of ‘dead’ (rejected) names
```

Comment:
Maybe we can add a short explanation about this? Like the one in the referenced source.

Reply:
@sstok Which explanation are you referring to? I'm afraid I can't find it...